package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "json-stream-analyzer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:JonathanMontane/json-stream-analyzer.git",
```

Comment:
new repo url is: https://github.com/algolia/json-stream-analyzer

Reply:
updated